spec/models/manageiq/providers/embedded_ansible/automation_manager_spec.rb

```diff
@@ -1,6 +1,4 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager do
-  it_behaves_like 'ansible automation_manager'
```

Comment:
I think this line should be removed based on what @jameswnl shared, but it probably makes sense to take it a step further and see what removing this line does:

https://github.com/ManageIQ/manageiq/blob/master/spec/spec_helper.rb#L25

Because it really shouldn't be needed in this repo any more.

Reply:
Good catch yeah that should be removed also